honeybee_radiance_postprocess/reader.py

```diff
@@ -0,0 +1,61 @@
+"""Post-processing reader functions."""
+
+import numpy as np
+import pyarrow.feather as feather
+
+from .util import binary_mtx_dimension
+
+
+def feather_to_numpy(filepath, pandas_dataframe=False):
+    """Read a feather file a NumPy array.
+    
+    Args:
+        filepath: Path to a feather file.
+        pandas_dataframe: A boolean to note if feather should read the file as Pandas
+            DataFrame. The function will still return a NumPy array, i.e., only reading
+            of the file is affected by this option. The Pandas DataFram will be converted
+            to NumPy.
+    
+    Returns:
+        A NumPy array.
+    """
+    if not pandas_dataframe:
+        # read file to PyArrow table
+        table = feather.read_table(filepath)
+        np_arrays = [arr.to_numpy() for arr in table]
+        data = np.array(np_arrays)
+    else:
+        # read file to Pandas DataFrame
+        dataframe = feather.read_feather(filepath)
+        data = dataframe.to_numpy().transpose()
+    
+    return data
+
+
+def binary_to_numpy(binary_file, nrows=None, ncols=None, ncomp=None):
```

Comment:
Since this library is Python 3 only it might make sense to start adding type hints to make it easier for people to use it.

```suggestion
def binary_to_numpy(binary_file: str, nrows: int = None, ncols : int = None, ncomp : int = None) -> np.Array:
```

Reply:
Added type hints to all the functions.